cypress/integration/i-can-enter-information.ts

```diff
@@ -0,0 +1,136 @@
+import { PageURLs } from "../../src/lib/urls";
+import { testData } from "./happy-path-test-data.spec";
+import {
+  andIClickContinue,
+  andIHaveSelected,
+  givenIAmAt,
+  givenIHaveBeenTo,
+  givenIHaveClicked,
+  givenIHaveClickedContinue,
+  givenIHaveSelected,
+  givenIHaveTyped,
+  thenTheUrlShouldContain,
+} from "./selectors-and-assertions.spec";
+
+export const givenIHaveEnteredMyBeaconDetails = (): void => {
```

Comment:
This function here is completing the check beacon details and beacon information. Do you think it's worth splitting it out into two functions of `givenIHaveEnteredCheckBeaconDetails` and `givenIHaveEnteredBeaconInformation` and then combining the two methods go give `givenIHaveEnteredMyBeaconDetails`

Reply:
Agreed.  I've gone with `givenIHaveFilledInCheckBeaconDetailsPage()` and `givenIHaveFiledInBeaconInformationPage()` to signify the lower level of abstraction of the constituent functions.